assessment_software/src/app/button-component/button.component.ts

```diff
@@ -0,0 +1,20 @@
+import { Component, EventEmitter, Input, Output } from '@angular/core';
+
+@Component({
+  selector: 'app-button',
+  templateUrl: './button.component.html',
+  styleUrls: ['./button.component.css'],
+})
+export class ButtonComponent {
+  @Input()
+  id!: string;
+  @Input() variant: 'Standard' | 'Filled' | 'Outlined' = 'Standard';
```

Comment:
should there be another button variant? for a button with white background and gray text

Reply:
I was thinking about that and I was hoping one can change the color using the color attribute:

`<app-button variant="Standard" color="grey"> Standard</app-button>`